crates/factors/Cargo.toml

```diff
@@ -18,11 +18,12 @@ http = "1.1.0"
 http-body-util = "0.1.2"
 serde_json = "1.0"
 spin-componentize = { path = "../componentize" }
-spin-factors-derive = { path = "../factors-derive", features = ["expander"] }
+# spin-factors-derive = { path = "../factors-derive", features = ["expander"] }
```

Comment:
Why is this commented out?

Reply:
This was a temporary fix to not see squiggly red error lines in the test file 
[`#[derive(RuntimeFactors)]`](https://github.com/fermyon/spin/pull/2655/files#diff-e42052e0e37b2c3a8ec952e6c4ded219d088bee335d8317a5af8d252c174a91bR10). 

Will change it back. Not sure if there is a better way to handle it now @lann

Reply:
It _should_ be fixed on the `factors` branch now.